src/components/input/Input.js

```diff
@@ -28,6 +28,7 @@ const InputNext = function Input({
 
   ...htmlAttributes
 }) {
+  // @ts-expect-error - "aria-label" is missing from HTMLInputAttributes
```

Comment:
An alternative solution here would be to cast `htmlAttributes` to some other type first.